### DISCUSSION TITLE
第3节课作业(19班)

### DISCUSSION THREAD

**Discussion by LYMDLUT** [InternLM/Tutorial#246] [作业]
第3节课作业(19班)

**Comment by hu-qi** [comment]
- 基础作业：

复现课程知识库助手搭建过程 (截图)

词向量模型下载
![image](https://github.com/InternLM/tutorial/assets/17986122/baba50ee-399b-4e59-be60-c33efac3058b)

载 NLTK 相关资源
![image](https://github.com/InternLM/tutorial/assets/17986122/bd40dad4-a27d-47b4-add1-e59e013d0710)

数据收集(以 InternLM 开源相关资料为例)
![image](https://github.com/InternLM/tutorial/assets/17986122/1c20af31-cea4-4ba6-95ff-8df4f35c8ac3)


创建并持久化向量数据库
![image](https://github.com/InternLM/tutorial/assets/17986122/dd11a75e-b380-45ec-9f5a-b2542be866f6)

运行gradio
![image](https://github.com/InternLM/tutorial/assets/17986122/0b488cc5-8025-48fd-9795-24b9850acbc6)





~~进阶作业：~~

~~选择一个垂直领域，收集该领域的专业资料构建专业知识库，并搭建专业问答助手，并在 [OpenXLab](https://openxlab.org.cn/apps) 上成功部署（截图，并提供应用地址）~~

**Comment by Jiangzzzzl** [comment]
<img width="843" alt="1" src="https://github.com/InternLM/tutorial/assets/150671792/82d9fcb2-8e5c-4af2-8185-0ce117dbc19a">
<img width="866" alt="2" src="https://github.com/InternLM/tutorial/assets/150671792/6c4b5824-7237-4bd4-816e-731eab2eb99b">
<img width="882" alt="3" src="https://github.com/InternLM/tutorial/assets/150671792/b705b7b3-c53a-4211-9b3c-c70a5c6cf321">
<img width="882" alt="4" src="https://github.com/InternLM/tutorial/assets/150671792/4c7a2196-e001-461b-b879-23a0e9a1bc6a">
<img width="882" alt="5" src="https://github.com/InternLM/tutorial/assets/150671792/fff2c2fa-a0ac-469b-be02-15c08325fb44">
<img width="882" alt="6" src="https://github.com/InternLM/tutorial/assets/150671792/4d34673f-8435-40fa-a158-5a321810d422">
<img width="882" alt="7" src="https://github.com/InternLM/tutorial/assets/150671792/eb4da9be-7a58-4217-8acc-f6ff9d53fd3d">
<img width="878" alt="8" src="https://github.com/InternLM/tutorial/assets/150671792/75259c39-f021-4ce9-a98f-cccec9c6e4cc">
<img width="878" alt="9" src="https://github.com/InternLM/tutorial/assets/150671792/41f0dea9-a5ec-47e0-af3e-ec5abe50cc64">
<img width="878" alt="10" src="https://github.com/InternLM/tutorial/assets/150671792/6b3cec34-3f71-48e2-9e50-c316dfee8e43">
<img width="878" alt="11" src="https://github.com/InternLM/tutorial/assets/150671792/6ec314ea-d654-4c49-b418-59673eebe6c9">
<img width="878" alt="12" src="https://github.com/InternLM/tutorial/assets/150671792/191b3e04-45cc-4108-93f6-9f77d21dc7dd">
<img width="878" alt="13" src="https://github.com/InternLM/tutorial/assets/150671792/a8c1ba19-ca6f-4adf-bcae-4ba52531cd14">
<img width="878" alt="14" src="https://github.com/InternLM/tutorial/assets/150671792/27b39191-19ee-4dd9-be7c-03bd500bc0e5">
<img width="1280" alt="15" src="https://github.com/InternLM/tutorial/assets/150671792/cf8c72f3-f37f-4771-a3f0-a5a8c2df0abc">

**Comment by melonwine** [comment]
https://github.com/melonwine/InternLMPlay/blob/master/03/asgmt03.md

**Comment by jts250** [comment]
https://github.com/jts250/homework_llm/blob/main/homework2.pdf

**Comment by AllYoung** [comment]
https://blog.csdn.net/sunshine_youngforyou/article/details/135538786

**Comment by romanwin28** [comment]
基础作业：

1、
<img width="918" alt="微信截图_20240111184226" src="https://github.com/InternLM/tutorial/assets/146464537/9cb06330-6fbc-468c-a829-1cf236c57941">
<img width="923" alt="微信截图_20240111184759" src="https://github.com/InternLM/tutorial/assets/146464537/27673362-342d-4762-a341-19cd700a19a4">

**Comment by JankinTian** [comment]
https://blog.csdn.net/xiao_xian_/article/details/135586301

**Comment by Sixdes** [comment]
https://github.com/Sixdes/puyu-course/tree/main/homework0112

**Comment by yunlong96** [comment]
第三节课作业：
![image](https://github.com/InternLM/tutorial/assets/53212855/32f9f093-d90e-4121-b9ea-3f42f3fc9601)
![image](https://github.com/InternLM/tutorial/assets/53212855/fd19a1fe-cc94-430b-b6e1-40a73a1ace4c)

**Comment by haohaocodeli** [comment]
http://t.csdnimg.cn/drF5P

**Comment by mkdirmushroom** [comment]
[第3节课作业](https://github.com/mkdirmushroom/note/blob/main/InternLM/tutorial2_assignment.md)

**Comment by Hamasuta666** [comment]
第三节课笔记与作业（作业在文档末）：https://zhuanlan.zhihu.com/p/680574294
